src/main/java/Validation.java

```diff
@@ -65,6 +65,11 @@ public static Integer parseInt(String s) {
         }
     }
 
+
+    public static String sanitizeYAML(final String s) {
+        return s.replace("\t", " ")
```

Comment:
Might need a `;`

Reply:
Whoops, I guess I've been writing too much golang these days

Reply:
I feel you :)